lib/commands/unpublish.js

```diff
@@ -71,7 +83,7 @@ class Unpublish extends BaseCommand {
 
     if ((!spec || !spec.rawSpec) && !force) {
       throw this.usageError(
-        'Refusing to delete entire project.\n' +
```

Comment:
Was there a compelling reason to remove this carriage return??

Reply:
yes, please revert it.

Reply:
@wraithgar ready.